karspexet/ticket/tests/test_views.py

```diff
@@ -1,25 +1,51 @@
 # coding: utf-8
 from django.shortcuts import reverse
-from django.test import TestCase
+from django.test import TestCase, RequestFactory
 from django.utils import timezone
 
 from karspexet.show.models import Show, Production
 from karspexet.ticket import views
 
 from karspexet.venue.models import Venue, SeatingGroup
 
+import pytest
+
 
 class TestHome(TestCase):
-    def test_home_lists_upcoming_shows(self):
+    def setUp(self):
+        rf = RequestFactory()
+        self.request = rf.get(reverse(views.home))
+
+
+    def test_home_lists_visible_upcoming_shows(self):
         venue = Venue.objects.create(name="Teater 1")
         production = Production.objects.create(name="Uppsättningen")
-        date = timezone.now()+timezone.timedelta(days=1)
+        date = timezone.now() + timezone.timedelta(days=1)
```

Comment:
Please reuse the same timezone.now to be more tidy. I also can't help to feel it will mess something up down the road. (Well at least the habit of calling timezone.now multiple times instead of caching the value.)